src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -43,6 +43,14 @@ const WSTRING agent_port = "DD_TRACE_AGENT_PORT"_W;
 // Sets the "env" tag for every span.
 const WSTRING env = "DD_ENV"_W;
 
+// Sets the "version" tag for every span.
+const WSTRING version = "DD_VERSION"_W;
```

Comment:
We don't want the `version` on every single span. For spans that are created under a DB service, the `version` would suggest that it's the version of that DB.

In other tracers we've stuck to the rule of setting the version in a span if and only if the span's service matches the globally configured service name.

Reply:
@jdgumz Is the desired outcome that only the root span is decorated with this?

Reply:
The root span and any other span that has a service name == the globally configured service.

Reply:
When `DD_VERSION` is set, the version tag is now set on all the main service's spans, but not on external service spans. Any global tags set via `DD_TAGS` get set on _all_ spans.